tests/unit/lms/services/h_api_test.py

```diff
@@ -16,42 +15,39 @@
 from lms.services.h_api import HAPI
 from lms.values import HUser
 
-BASE_URL = "http://example.com/base/"
-AUTHORITY = "test-authority"
-
 
 class TestHAPI:
 
     # We're accessing h_api._api_request a lot in this test class, so disable
     # protected-access messages.
     # pylint: disable=protected-access
 
-    def test_get_user_works(self, h_api):
-        h_api._api_request.return_value.json.return_value = {
+    def test_get_user_works(self, h_api, _api_request):
+        _api_request.return_value.json.return_value = {
             "display_name": sentinel.display_name
         }
 
         user = h_api.get_user("username")
 
-        h_api._api_request.assert_called_once_with(
-            "GET", path="users/acct:username@test-authority"
+        _api_request.assert_called_once_with(
+            "GET", path="users/acct:username@TEST_AUTHORITY"
```

Comment:
The value of authority has changed because we're now using the global `pyramid_request` fixture instead of the test module creating its own similar-but-not-quite-the-same `p_request` fixture